silkrpc/main.cpp

```diff
@@ -167,6 +167,8 @@ int main(int argc, char* argv[]) {
         };
 
         // Check protocol version compatibility with Core Services
+        SILKRPC_LOG << "Checking protocol version compatibility with core services...\n";
+
```

Comment:
in context_pool pass wait_mode selected by user

Reply:
Agreed. I would leave this for a dedicated PR, you could open an issue just for tracking

Reply:
add issue #280